### DISCUSSION TITLE
Customizing the swagger documentation

### DISCUSSION THREAD

**Discussion by dheerajthodupunoori** [fastapi/fastapi#3448] [Questions] [answered]
Hi team ,

I have an API developed using FastAPI , just want to know how we can customize my swagger documentation and also how can i direct my application route to swagger doc when launched?

Thanks in Advance

**Comment by elrik75** [comment, marked as answer]
Look at this: https://fastapi.tiangolo.com/tutorial/metadata/

**Comment by dheerajthodupunoori** [comment]
Thanks @elrik75
